core/project_settings.cpp

```diff
@@ -916,7 +916,7 @@ void ProjectSettings::_add_property_info_bind(const Dictionary &p_info) {
 
 	PropertyInfo pinfo;
 	pinfo.name = p_info["name"];
-	ERR_FAIL_COND(!props.has(pinfo.name));
+	ERR_FAIL_COND(props.has(pinfo.name));
```

Comment:
That seems bogus. This method calls `set_custom_property_info`, which has the exact same check, and for a good reason, as `custom_prop_info` is indexed by `pinfo.name` (I assume it means this code expects `props` and `custom_prop_info` to have the same keys).

This change should just remove one error to then fail further up the call stack.